doc/reference/kernel/index.rst

```diff
@@ -126,3 +126,4 @@ These pages cover other kernel services.
    other/version.rst
    other/fatal.rst
    other/thread_local_storage.rst
+   other/event_manager.rst
```

Comment:
Why are you adding this under kernel/other? This should be alongside other subsystems maybe? The other/ under kernel is historical and needs some cleanup, so lets not add more things here.

Reply:
Thank you for your review. Could you point me where should this documentation in zephyr be placed?

Reply:
Please place it in doc/reference/event_manager